plugins/aws/sagemaker/notebookCmkEncrypted.js

```diff
@@ -0,0 +1,120 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Notebook Instance CMK Encrypted',
+    category: 'AI & ML',
+    domain: 'Compute',
+    severity: 'High',
+    description: 'Ensure Notebook instances are encrypted with desired encryption level.',
+    more_info: 'When you encrypt AWS Sagemaker notebook instance using your own AWS Customer Managed Keys (CMKs) for enhanced protection, you have full control over who can use the encryption keys to access your notebook data.',
```

Comment:
```suggestion
    more_info: 'AWS Sagemaker notebook instances encryption should have data encryption enabled with KMS Customer Master Key (CMK) instead of AWS-managed Key for cross-account access and in order to gain more granular control over encryption/decryption process.',
```